migrations/2023-05-10-095739_force_enable_undetermined_language/up.sql

```diff
@@ -0,0 +1,7 @@
+-- force enable undetermined language for all users
```

Comment:
You don't need three queries here. Just do `insert into ... on conflict (local_user_id, language_id) ignore`

https://stackoverflow.com/questions/1009584/how-to-emulate-insert-ignore-and-on-duplicate-key-update-sql-merge-with-po